src/Styleguide/Pages/Artwork/Components/ArtworkSidebar/ArtworkSidebarSizeInfo.tsx

```diff
@@ -0,0 +1,43 @@
+import { Serif } from "@artsy/palette"
+import React from "react"
+import { createFragmentContainer, graphql } from "react-relay"
+import { Box } from "Styleguide/Elements/Box"
+
+import { ArtworkSidebarSizeInfo_piece } from "__generated__/ArtworkSidebarSizeInfo_piece.graphql"
```

Comment:
Why this rename from artwork to `_piece` was necessary here, because it is present in both artwork and edition set? In Gravity the interface is 'Dimensions'.... So maybe it can be named it here as well? Though it includes both dimensions and edition info. Maybe 'sizable' or 'measurable'. 'Piece' is so generic:)

Reply:
The interface for them includes more data than just dimensions, such as `edition_of`, see https://github.com/artsy/metaphysics/pull/1140 for the full interface.

Seeing as the component now _is_ generic I went with a more generic name :) I’m open to other suggestions if there are better ones.